static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner-parameters.js

```diff
@@ -70,10 +54,25 @@ export const getControlEngagementBannerParams = (): Promise<?EngagementBannerTem
                     }. Stack: ${err.stack}`
                 ),
                 {
-                    feature: 'engagement-banner-test',
+                    feature: 'engagement-banner',
                 },
                 false
             );
 
+            // The banner tests work by overriding built-in parameters.
+            // So the default case is override nothing.
+            // As opposed to the epic where we return default copy.
+            // TODO: but could we return defaultEngagementBannerParams here?
```

Comment:
There's a question in here for @ionamckendrick and @jessewilkins 

The only thing stopping us handling the fallback case here is that the double banner (https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/ui/first-pv-consent-plus-engagement-banner.js) relies on `defaultEngagementBannerParams`.

That is to say, the double banner only ever uses hardcoded copy, never anything from the Google Doc.

It would be great if it used the copy from the Google Doc since a lot of people see it and the hardcoded copy will get stale quickly (it's not a worst-case fallback seen by very few people, as in the case of the epic). But my question is, do we usually check the double banner on iPhone 5 when we change the control copy? I think we left it hardcoded because we were worried about the text growing and taking over the screen when combined with the cookie banner.

Reply:
Having chatted to @ionamckendrick, it seems we want the double banner to behave the same as the single banner - respecting banner tests and the banner Google Doc copy.

This is probably also relevant to campaign work with @johnduffell - if the double-banner automatically respects tests (hard-coded or Google Sheet), there's no extra hackery needed to get the custom campaign banner to display with the cookie banner. We just create a test with a custom template.

Reply:
Going to look at that in a separate PR